packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -59,3 +59,16 @@ def test_ip_per_container(cluster):
         app_port = app_definition['container']['docker']['portMappings'][0]['containerPort']
         cmd = '/opt/mesosphere/bin/curl -s -f http://{}:{}/ping'.format(service_points[1].ip, app_port)
         ensure_routable(cmd, service_points)()
+
+
+def test_ip_per_container_with_named_vip(cluster):
+    """Test if we are able to connect to a task with ip-per-container mode using named vip
+    """
+    app_definition, test_uuid = cluster.get_test_app_in_docker(ip_per_container=True)
+    app_definition['container']['docker']['portMappings'][0]['servicePort'] = 6000
```

Comment:
We need to delete the hostPort and servicePort entry to prevent port mapping from occuring.